src/main/resources/liquibase/workbench_changelog/15_4_0.xml

```diff
@@ -81,4 +81,38 @@
 		</sql>
 	</changeSet>
 
+    <changeSet author="nahuel" id="v15.4.0-1">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(1) from workbench_tool where name = 'graphical_queries';
+            </sqlCheck>
+        </preConditions>
+        <comment>Move Graphical filtering and Study Comparison tools to a new "Graphical Queries" sub-menu</comment>
+        <sql dbms="mysql" splitStatements="true">
+            # Create new tool and side menu
+            insert into workbench_tool(name, group_name, title, version, tool_type, path)
+            values ('graphical_queries', 'graphical_queries', 'Graphical Queries', '15.4', 'WEB', '/ibpworkbench/controller/graphical-queries');
+            insert into workbench_sidebar_category_link(tool_name, sidebar_category_id, sidebar_link_name, sidebar_link_title)
+            values ('graphical_queries', 2, 'graphical_queries', 'Graphical Queries');
+
+            # Create new permission
+            set @information_management_id = (select permission_id from permission where name = 'INFORMATION_MANAGEMENT');
+            set @sidebar_category_link_id = (select sidebar_category_link_id from workbench_sidebar_category_link where tool_name = 'graphical_queries');
+
+            insert into permission(name, description, parent_id, workbench_sidebar_category_link_id)
+            values ('GRAPHICAL_QUERIES', 'Graphical Queries', @information_management_id, @sidebar_category_link_id);
+            set @last_permission_id = (select last_insert_id());
+            insert into role_type_permission(role_type_id, permission_id, selectable) values (1, @last_permission_id, 1);
+            insert into role_type_permission(role_type_id, permission_id, selectable) values (2, @last_permission_id, 1);
+            insert into role_type_permission(role_type_id, permission_id, selectable) values (3, @last_permission_id, 1);
+
+            # associate new 'GRAPHICAL_QUERIES' permission to every role that has 'BROWSE_STUDIES'
```

Comment:
Is it needed when the role already contains INFORMATION_MANAGEMENT permission? Seems like in that case we should not inset a new role_permission  @nahuel-soldevilla

Reply:
@clarysabel good point, but I think in this case it's selecting only the roles that specifically has BROWSE_STUDIES, so if any role has INFORMATION_MANAGEMENT it probably won't have BROWSE_STUDIES (unless some manual db manipulation has taken place)
Right? let me know if I'm missing something

Reply:
Rigth, they should not have both BROWSE_STUDIES and INFORMATION_MANAGEMENT unless someone manually changed the data